2022-02-23 11:09:56,558 INFO    Thread-8  :1698414 [wandb_setup.py:_flush():75] Loading settings from /home/eunchan/.config/wandb/settings
2022-02-23 11:09:56,558 INFO    Thread-8  :1698414 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-02-23 11:09:56,558 INFO    Thread-8  :1698414 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'entity': 'eckim', 'project': 'sweeps-random', 'root_dir': '/home/eunchan/Desktop/Study/Wandb-sweeps', 'run_id': 'k7a109g6', 'sweep_param_path': '/home/eunchan/Desktop/Study/Wandb-sweeps/wandb/sweep-233dsju3/config-k7a109g6.yaml', 'sweep_id': '233dsju3'}
2022-02-23 11:09:56,558 INFO    Thread-8  :1698414 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program': 'train.py'}
2022-02-23 11:09:56,558 INFO    Thread-8  :1698414 [wandb_init.py:_log_setup():386] Logging user logs to /home/eunchan/Desktop/Study/Wandb-sweeps/wandb/run-20220223_110956-k7a109g6/logs/debug.log
2022-02-23 11:09:56,559 INFO    Thread-8  :1698414 [wandb_init.py:_log_setup():387] Logging internal logs to /home/eunchan/Desktop/Study/Wandb-sweeps/wandb/run-20220223_110956-k7a109g6/logs/debug-internal.log
2022-02-23 11:09:56,559 INFO    Thread-8  :1698414 [wandb_init.py:init():420] calling init triggers
2022-02-23 11:09:56,559 INFO    Thread-8  :1698414 [wandb_init.py:init():425] wandb.init called with sweep_config: {'batch_size': 58, 'dropout': 0.5, 'epochs': 5, 'learning_rate': 0.07110574996085128, 'optimizer': 'adam'}
config: {'epochs': 2, 'batch_size': 128, 'fc_layer_size': 128, 'weight_decay': 0.0005, 'learning_rate': 0.001, 'activation': 'relu', 'optimizer': 'adam', 'seed': 42}
2022-02-23 11:09:56,559 INFO    Thread-8  :1698414 [wandb_init.py:init():471] starting backend
2022-02-23 11:09:56,559 INFO    Thread-8  :1698414 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-23 11:09:56,561 INFO    Thread-8  :1698414 [backend.py:ensure_launched():219] starting backend process...
2022-02-23 11:09:56,593 INFO    Thread-8  :1698414 [backend.py:ensure_launched():224] started backend process with pid: 1698602
2022-02-23 11:09:56,594 INFO    Thread-8  :1698414 [wandb_init.py:init():480] backend started and connected
2022-02-23 11:09:56,594 INFO    Thread-8  :1698414 [wandb_run.py:_config_callback():966] config_cb None None {'batch_size': 58, 'dropout': 0.5, 'epochs': 5, 'learning_rate': 0.07110574996085128, 'optimizer': 'adam'}
2022-02-23 11:09:56,598 INFO    Thread-8  :1698414 [wandb_init.py:init():550] updated telemetry
2022-02-23 11:09:56,599 INFO    Thread-8  :1698414 [wandb_init.py:init():581] communicating current version
2022-02-23 11:09:57,002 INFO    Thread-8  :1698414 [wandb_init.py:init():586] got version response 
2022-02-23 11:09:57,002 INFO    Thread-8  :1698414 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-23 11:09:57,242 INFO    Thread-8  :1698414 [wandb_init.py:init():624] starting run threads in backend
2022-02-23 11:09:58,665 INFO    Thread-8  :1698414 [wandb_run.py:_console_start():1827] atexit reg
2022-02-23 11:09:58,666 INFO    Thread-8  :1698414 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-02-23 11:09:58,666 INFO    Thread-8  :1698414 [wandb_run.py:_redirect():1706] Redirecting console.
2022-02-23 11:09:58,666 ERROR   Thread-8  :1698414 [wandb_run.py:_redirect():1765] Failed to redirect.
Traceback (most recent call last):
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1758, in _redirect
    out_redir.install()
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/eunchan/miniconda3/envs/python38/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-02-23 11:09:58,666 INFO    Thread-8  :1698414 [wandb_init.py:init():651] run started, returning control to user process
2022-02-23 11:09:58,702 INFO    Thread-8  :1698414 [wandb_watch.py:watch():43] Watching
2022-02-23 11:10:05,102 INFO    Thread-8  :1698414 [wandb_run.py:finish():1468] finishing run eckim/sweeps-random/k7a109g6
2022-02-23 11:10:05,102 INFO    Thread-8  :1698414 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-23 11:10:05,102 INFO    Thread-8  :1698414 [wandb_run.py:_restore():1769] restore
2022-02-23 11:10:05,117 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 535
}

2022-02-23 11:10:05,465 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 535
}

2022-02-23 11:10:06,249 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 36511
}

2022-02-23 11:10:06,352 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 37203
}

2022-02-23 11:10:06,482 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 535
  total_bytes: 37203
}

2022-02-23 11:10:06,585 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 37203
  total_bytes: 37203
}

2022-02-23 11:10:06,688 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 37203
  total_bytes: 37203
}

2022-02-23 11:10:06,790 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 37203
  total_bytes: 37203
}

2022-02-23 11:10:06,894 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 37203
  total_bytes: 37203
}

2022-02-23 11:10:06,997 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 37203
  total_bytes: 37203
}

2022-02-23 11:10:07,101 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 37203
  total_bytes: 37203
}

2022-02-23 11:10:08,032 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 37203
  total_bytes: 37203
}

2022-02-23 11:10:08,546 INFO    Thread-8  :1698414 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 37203
  total_bytes: 37203
}
local_info {
}

2022-02-23 11:10:09,778 INFO    Thread-8  :1698414 [wandb_run.py:_append_history():2144] rendering history
2022-02-23 11:10:09,778 INFO    Thread-8  :1698414 [wandb_run.py:_append_summary():2102] rendering summary
2022-02-23 11:10:09,778 INFO    Thread-8  :1698414 [wandb_run.py:_append_files():2194] logging synced files
